backend/api/grants/mutations.py

```diff
@@ -143,6 +145,23 @@ class UpdateGrantInput(BaseGrantInput):
 )
 
 
+@strawberry.input
+class SendGrantReplyInput:
+    instance: strawberry.ID
+    status: Status
```

Comment:
technically here we should allow a subset of statuses, otherwise, someone can switch to pending if they want?

Reply:
I added a if later, but this will be more secure I think